docs/src/_getting-started/part3.md

```diff
@@ -0,0 +1,176 @@
+</div>
+
+## Select the author data from GitHub
+Just as in Part 1, you can use the [`meltano select`](/reference/command-line-interface#select) command to select additional data from the source. We're simply going to select all "commits" data:
+
+```bash
+meltano select tap-github commits "*"
+```
+
+This will add the following line to your project file:
+```yaml
+      extractors:
+      - name: tap-github
+        select:
+        - commits.url # <== technically not necessary anymore, but no need to delete
+        - commits.sha # <== technically not necessary anymore, but no need to delete
+        - commits.* # <== new data.
+```
+
+You can test that the new data is extracted by using `meltano invoke`:
+
+<div class="termy">
+```console
+$ meltano invoke tap-github
+2022-09-22T07:36:52.985090Z [info     ] Environment 'dev' is active
+{"type": "STATE", "value":  [...]}
+INFO Starting sync of repository: sbalnojan/meltano-example-el
+---> 100%
+{"type": "SCHEMA", "stream": "commits", [...]
+
+INFO METRIC: {"type": "timer", "metric":  [...]
+
+{"type": "RECORD", "stream": "commits", "record": {"sha": "c771a832720c0f87b3ce53ac12bdcbf742df4e3d", "commit": {"author": {"name": "Horst", "email":
+[...]
+"sbalnojan/meltano-example-el"}, "time_extracted": "2022-09-22T07:37:06.289545Z"}
+
+...[many more records]...
+
+{"type": "STATE", "value": {"bookmarks": {"sbalnojan/meltano-example-el": {"commits": {"since": "2022-09-22T07:37:06.289545Z"}}}}}
+´´´
+</div>
+
+## Installing the transform-field mapper
+To add inline transformations, we need a new plugin. We're going to use the mapper "transform-field". To add this plugin, use the `meltano add mapper` command:
+
+<div class="termy">
+```console
+$ meltano add mapper transform-field
+
+Added mapper 'transform-field' to your Meltano project
+Variant:        transferwise (default)
+Repository:     https://github.com/transferwise/pipelinewise-transform-field
+Documentation:  https://hub.meltano.com/mappers/transform-field
+
+Installing mapper 'transform-field'...
+---> 100%
+Installed mapper 'transform-field'
+
+To learn more about mapper 'transform-field', visit https://hub.meltano.com/mappers/transform-field
+</div>
+
+We're now going to add two mappings to this mapper.
+
+## Adding a emails-hidden mapping
+To add our first mapping, we're going to edit the `meltano.yml` file located inside your root project directory. Modify the block for the `transform-field` mapper as shown below:
+
+```yaml
+mappers:
+  - name: transform-field
+    variant: transferwise
+    pip_url: pipelinewise-transform-field
+    executable: transform-field
+   mappings:
+    - name: hide-github-mails
+      config:
+        transformations:
+          - field_id: "commit"
+            tap_stream_name: "commits"
+            field_paths: ["author/email", "committer/email"]
+            type: "MASK-HIDDEN"
```

Comment:
@sbalnojan and @tayloramurphy - Separate from my prior comment proposing we add mappings _after_ the "T" in "ELT":

Whereever we put the content here (and I agree it is valuable), I think it's probably best if we promote the `meltano-map-transfomer` plugin, which we own - and not point people in the direction of `transform-field`. The rationale would be: (1) our plugin has slightly better features, (2) for SDK-based taps and targets, the config will basically be drop-in compatible across taps, targets, and the dedicated mapper, and (3) we don't have ability to improve the `field-transform` plugin when people run into issues, since we aren't maintainers on it.

Reply:
That was my plan, but quite frankly, I wasn't able to get it to work after a full day, so I had to opt for what works out of the box (mostly), even this mapper is IMHO way below average in terms of usability. 

So I propose:
- we push it out as it is
- someone adds docs to the meltano-map-transformer (I already highlighted that on a separate issue)
- we swap it out afterwards.